src/components/Profile.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom';
+
+class Profile extends Component {
+  render() {
+    return (
+      <form>
+        <h1>My Profile</h1>
+        <h2>Basic Information</h2>
+        <p>Name : Gopi Krishna</p>
```

Comment:
We should probably make these variables instead of plaintext so they're easier to change with the values loaded from the database later.